mapbox/app/src/main/java/com/mapbox/services/android/testapp/location/GoogleLocationEngine.java

```diff
@@ -88,7 +87,7 @@ public void onConnectionFailed(@NonNull ConnectionResult connectionResult) {
 
   @Override
   public Location getLastLocation() {
-    if (googleApiClient.isConnected() && PermissionsManager.areLocationPermissionsGranted(context.get())) {
+    if (googleApiClient.isConnected()) {
```

Comment:
`PermissionsManager.areLocationPermissionsGranted(context.get()` is broken for pre Marshmallow devices